src/components/Modals/Send/utils.ts

```diff
@@ -242,20 +254,33 @@ export const handleSend = async ({
       const contractAddress = contractAddressOrUndefined(asset.assetId)
       const fees = estimatedFees[feeType] as FeeData<KnownChainIds.SolanaMainnet>
 
+      const solanaAdapter = assertGetSolanaChainAdapter(chainId)
+      const { account } = fromAccountId(sendInput.accountId)
+      const instructions = await solanaAdapter.buildEstimationInstructions({
+        from: account,
+        to,
+        tokenId: contractAddress,
+        value,
+      })
+
       const input: BuildSendTxInput<KnownChainIds.SolanaMainnet> = {
         to,
         value,
         wallet,
         accountNumber: bip44Params.accountNumber,
-        chainSpecific: {
-          tokenId: contractAddress,
-          computeUnitLimit: fees.chainSpecific.computeUnits,
-          computeUnitPrice: fees.chainSpecific.priorityFee,
-        },
+        chainSpecific:
+          instructions.length <= 1
```

Comment:
<= 1 vs. === 0 or !instructions is on purpose here - when we hit this, we have added one instruction already in `buildEstimationInstructions()` i.e the system one